libs/data-mapper/src/lib/utils/Connection.Utils.ts

```diff
@@ -10,8 +10,15 @@ import { addReactFlowPrefix, addTargetReactFlowPrefix } from './ReactFlow.Util';
 import { isSchemaNodeExtended } from './Schema.Utils';
 import type { WritableDraft } from 'immer/dist/internal';
 
-// NOTE: This method should be the gateway for anything getting into dataMapConnections
-// - meaning all default inputs/etc can safely be managed in this singular spot
+/**
+ * Creates a connection entry in the connections dictionary if it doesn't already exist.
+ * This is the gateway for any entries made into dataMapConnections - meaning all default
+ * inputs/etc can safely be managed in this singular spot.
+ *
+ * @param connections The connections dictionary to add the entry to
+ * @param node The node to create the connection entry for
+ * @param reactFlowKey The reactFlowKey of the node
+ */
 export const createConnectionEntryIfNeeded = (
```

Comment:
I see in a lot of spots, you've deleted calls to this method, is it still necessary to be exported? It might be even if there are no calls to it. Thoughts?